index.html

```diff
@@ -0,0 +1,25 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="style.css" />
+    <title>Document</title>
+  </head>
+  <body>
+    <navbar id="navbar">
+      <a href="#projects">Projects</a>
+      <a href="#">About</a>
+      <a id="profile-link" target="_blank">my github</a>
+    </navbar>
+    <section id="welcome-section">
+      <h1>Hey, We are Geeky & Jude</h1>
+    </section>
+
+    <section id="projects">
```

Comment:
Projects should be in main tag to have better semantics and make it more readable.

Reply:
> Projects should be in main tag to have better semantics and make it more readable.

- [ ] This also, I would implore you to be polite in your comment, in this case, it can be `I would implore you to... or I think it would be better...`. Nice one, you gave reason for your comment for student to also see from your point of view.